tsl/test/expected/cagg_policy.out

```diff
@@ -156,6 +280,284 @@ CREATE MATERIALIZED VIEW max_mat_view_date
+                                                                     show_policies                                                                      
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+ {"policy_name": "policy_refresh_continuous_aggregate", "refresh_interval": "@ 1 hour", "refresh_end_offset": null, "refresh_start_offset": "@ 5 days"}
+(1 row)
+
+--Cross policy checks
+-- Refresh and compression policies overlap
+SELECT timescaledb_experimental.add_policies('max_mat_view_date', compress_after => '20 days'::interval, drop_after => '25 days'::interval);
+ add_policies 
+--------------
+ t
+(1 row)
+
+SELECT timescaledb_experimental.alter_policies('max_mat_view_date', compress_after=> '4 days'::interval);
+ERROR:  refresh and compression policies overlap
+SELECT timescaledb_experimental.show_policies('max_mat_view_date');
+                                                                     show_policies                                                                      
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+ {"policy_name": "policy_compression", "compress_after": "@ 20 days", "compress_interval": "@ 35 days"}
+ {"policy_name": "policy_refresh_continuous_aggregate", "refresh_interval": "@ 1 hour", "refresh_end_offset": null, "refresh_start_offset": "@ 5 days"}
+ {"drop_after": "@ 25 days", "policy_name": "policy_retention", "retention_interval": "@ 1 day"}
+(3 rows)
+
+-- Refresh and retention policies overlap
+SELECT timescaledb_experimental.alter_policies('max_mat_view_date', refresh_start_offset =>'5 days'::interval, drop_after=> '4 days'::interval);
+ERROR:  refresh and retention policies overlap
+SELECT timescaledb_experimental.show_policies('max_mat_view_date');
+                                                                     show_policies                                                                      
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+ {"policy_name": "policy_compression", "compress_after": "@ 20 days", "compress_interval": "@ 35 days"}
+ {"policy_name": "policy_refresh_continuous_aggregate", "refresh_interval": "@ 1 hour", "refresh_end_offset": null, "refresh_start_offset": "@ 5 days"}
+ {"drop_after": "@ 25 days", "policy_name": "policy_retention", "retention_interval": "@ 1 day"}
+(3 rows)
+
+--Do not allow refreshed data to be deleted
+SELECT add_retention_policy('continuous_agg_max_mat_date', '25 days'::interval);
+ add_retention_policy 
+----------------------
+                 1024
+(1 row)
+
+SELECT timescaledb_experimental.alter_policies('max_mat_view_date', refresh_start_offset =>'25 days'::interval);
+ERROR:  do not allow refreshed data to be deleted
```

Comment:
We should probably rephrase this message, it doesn't read right.

Reply:
any suggestions...?